client/src/hu_elements.cpp

```diff
@@ -184,6 +184,23 @@ int teamTextColor(byte team) {
 	return color;
 }
 
+// generate an ordinal suffix from an integer (1 -> "st", 2 -> "nd", etc)
+// used in HelpText() to display position in queue
+static std::string ordinal(int n)
```

Comment:
This is nit-picky but can we just return a const char* here instead to avoid extra object construction.

Reply:
Yeah, that's fine. I think std::string was just a carry-over from my old queue code, where I was concatenating the return value with another string, but it looks like std::ostringstream eliminates the need for that.